src/lib/assets/helm.ts

```diff
@@ -207,13 +207,99 @@ export function admissionDeployTemplate(buildTimestamp: string) {
                   secretName: {{ .Values.uuid }}-api-token
               - name: module
                 secret:
-                  secretName: {{ .Values.uuid }}-module  
+                  secretName: {{ .Values.uuid }}-module
               {{- if .Values.admission.extraVolumes }}
               {{- toYaml .Values.admission.extraVolumes | nindent 8 }}
               {{- end }}
     `;
 }
 
+export interface ClusterRoleRule {
+  apiGroups: string[];
+  resources: string[];
+  verbs: string[];
+}
+
+export interface RoleRule {
+  apiGroups: string[];
+  resources: string[];
+  verbs: string[];
+}
```

Comment:
use upstream types, with kubernetes/client-node? 

These are the same type:

```plaintext
$ k explain clusterRole.rules
GROUP:      rbac.authorization.k8s.io
KIND:       ClusterRole
VERSION:    v1

FIELD: rules <[]PolicyRule>

$ k explain role.rules 
GROUP:      rbac.authorization.k8s.io
KIND:       Role
VERSION:    v1

FIELD: rules <[]PolicyRule>


```

Reply:
the interfaces are now gone